### PR TITLE
Bug 1205019 - [Music][NGA] When search bar is shown, songs cannot be tapped

### PR DESCRIPTION
**Pull request by justindarc**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=523dac773d4e261359eb60e4f1c45bfb008ad5db

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4cbc113b72964bb7fa471497aa50a4f32bb9dfdb

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=13eef3ab8f17e8ee3960e0df2bbbd3e8ba105cf6

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1e975de92bfa11256c0f94c277c8fdb662de1515

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=218af97146b9f3102649f3553d4254dd59137d16

File: dev_apps/music-nga/views/albums/index.html [hunk 1]
```diff
@@ -17,14 +17,22 @@
...
   <script src="/elements/music-search.js"></script>
 </head>
 <body role="application">
-  <music-search id="search"></music-search>
-  <gaia-fast-list id="list"></gaia-fast-list>
+  <gaia-fast-list id="list" offset="37">
+    <music-search id="search"></music-search>
+    <template>
+      <a href="/album-detail?id=${name}">
+        <h3>${metadata.album}</h3>
```
Comment (jimporter) [inline dev_apps/music-nga/views/albums/index.html]: Albums should have the artist as a subtitle, like in OGA.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6cf3c50b9fb551ea3659dc406f2e8f1b88bf46d1

File: dev_apps/music-nga/views/albums/index.html (hunk 1 above)
Comment (justindarc) [inline dev_apps/music-nga/views/albums/index.html]: Done.

**Pull request merged**